### PR TITLE
add info about ELK

### PR DESCRIPTION
**Pull request by emuus**

### CODE REVIEW

**Comment by AliceAlbano** [comment]
Thanks for the PR! I don't see any issue, but as I wrote a part of it, I added a new reviewer with fresh eyes to have another review :)

File: admin/en/source/pages/monitoring/elk.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+.. _elk:
+
+Configuring UForge to Send Logs to ELK
+---------------------------------------
+
+It is possible to visualize UForge logs in a dedicated tool: an ELK server (ElasticSearch, Logstash, Kibana). ELK centralizes all the logs and offers the possibility to navigate and search through the logs. It allows you to analyze and visualize the logs. If you have an ELK server installed, you can configure UForge so that UForge logs are sent to your ELK server.
+
+UForge logs retrieval can be configured as follows:
+
+  #. Edit the file ``/etc/UShareSoft/uforge/uforge.conf``. Change the value of ``FILEBEAT_SERVICE_ENABLED`` to ``true``.
```
Comment (ammarjohar) [inline admin/en/source/pages/monitoring/elk.rst]: /etc/filebeat/filebeat.yml

File: admin/en/source/pages/monitoring/elk.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.. Copyright 2018 FUJITSU LIMITED
+
+.. _elk:
+
+Configuring UForge to Send Logs to ELK
+---------------------------------------
+
+It is possible to visualize UForge logs in a dedicated tool: an ELK server (ElasticSearch, Logstash, Kibana). ELK centralizes all the logs and offers the possibility to navigate and search through the logs. It allows you to analyze and visualize the logs. If you have an ELK server installed, you can configure UForge so that UForge logs are sent to your ELK server.
+
```
Comment (ammarjohar) [inline admin/en/source/pages/monitoring/elk.rst]: please make it more clear that uforge.conf and filebeat.yml should be modified one all the nodes

File: admin/en/source/pages/monitoring/elk.rst (hunk 1 above)
Comment (ammarjohar) [reply admin/en/source/pages/monitoring/elk.rst]: sorry not this one

File: admin/en/source/pages/monitoring/elk.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+UForge logs retrieval can be configured as follows:
+
+  #. Edit the file ``/etc/UShareSoft/uforge/uforge.conf``. Change the value of ``FILEBEAT_SERVICE_ENABLED`` to ``true``.
+
+  #. Edit the file ``/tmpl/filebeat.yml.tmpl`` and put the IP and port of your ELK server in the section:
```
Comment (ammarjohar) [inline admin/en/source/pages/monitoring/elk.rst]: /etc/filebeat/filebeat.yml

File: admin/en/source/pages/monitoring/elk.rst (hunk 2 above)
Comment (emuus) [reply admin/en/source/pages/monitoring/elk.rst]: ok

File: admin/en/source/pages/monitoring/elk.rst (hunk 3 above)
Comment (emuus) [reply admin/en/source/pages/monitoring/elk.rst]: ok

**Pull request merged**